pkgs/development/python-modules/cffi/default.nix

```diff
@@ -13,15 +13,14 @@ if isPyPy then null else buildPythonPackage rec {
   propagatedBuildInputs = [ libffi pycparser ];
   buildInputs = [ pytest ];
 
-  patchPhase = ''
-    substituteInPlace testing/cffi0/test_ownlib.py --replace "gcc" "cc"
```

Comment:
ohh, only see just now that this is removed. Not so sure about that. Will also cause quite a rebuild.

Reply:
pretty sure the NIX_CFLAGS_COMPILE attribute also causes a rebuild

Reply:
even when optional? Hmm, yea I think you're right.